src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -539,6 +536,11 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
             } else {
                 authenticateAndSelectDB();
                 LOG.info("Reconnected to Redis");
+                try {
+                    loadRedisScripts();
+                } catch (IOException e) {
+                    LOG.error("Failed to load redis scripts", e);
```

Comment:
Maybe `Failed to reload Lua scripts after reconnect`

Reply:
done